### PR TITLE
setting up cypress

### PR DESCRIPTION
**Pull request by acruzgon**
This PR is the first approach of adding the setup for Cypress tests.
[AAP-15094](https://issues.redhat.com/browse/AAP-15094)

### CODE REVIEW

File: cypress/e2e/lightspeed-org-admin.cy.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+describe('Navigate to the Seats Administration page as lightspeed-test-user', () => {
```
Comment (lahavyuv86) [inline cypress/e2e/lightspeed-org-admin.cy.ts:1]: I description should be "as lightspeed-org-admin" no?

File: cypress/e2e/lightspeed-test-user-2.cy.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+describe('Navigate to the Seats Administration page as lightspeed-test-user', () => {
```
Comment (lahavyuv86) [inline cypress/e2e/lightspeed-test-user-2.cy.ts:1]: also here: lightspeed-test-user-2

File: cypress/e2e/lightspeed-org-admin.cy.ts (hunk 1 above)
Comment (acruzgon) [reply cypress/e2e/lightspeed-org-admin.cy.ts]: yes - just fixed, thanks :D

File: cypress/e2e/lightspeed-test-user-2.cy.ts (hunk 2 above)
Comment (acruzgon) [reply cypress/e2e/lightspeed-test-user-2.cy.ts]: thanks! just fixed as well

**Review by lahavyuv86 — APPROVED** [review]
LGTM

**Pull request merged**